os_brick/initiator/connectors/lightos.py

```diff
@@ -124,21 +122,19 @@ def dsc_do_connect_volume(self, connection_info):
             dscfile.flush()
             try:
                 dest_name = self.dsc_file_name(uuid)
-                cmd = shlex.split('mv {} {}'.format(dscfile.name, dest_name))
-                (out, err) = self._execute(*cmd, root_helper=self._root_helper,
-                                           run_as_root=True)
-            except putils.ProcessExecutionError as e:
-                LOG.warning("LIGHTOS: Failed to run command {}".format(cmd))
+                priv_lightos.move_dsc_file(dscfile.name, dest_name)
+            except Exception as e:
+                LOG.warning(
+                    "LIGHTOS: Failed to create dsc file for connection with"
+                    f" uuid:{uuid}")
                 raise e
 
     def dsc_disconnect_volume(self, connection_info):
         uuid = connection_info['uuid']
         try:
-            cmd = shlex.split('rm -f {}'.format(self.dsc_file_name(uuid)))
-            (out, err) = self._execute(*cmd, root_helper=self._root_helper,
-                                       run_as_root=True)
-        except putils.ProcessExecutionError as e:
-            LOG.warning("LIGHTOS: Failed to run command {}".format(cmd))
+            priv_lightos.delete_dsc_file(self.dsc_file_name(uuid))
+        except Exception as e:
+            LOG.warning("LIGHTOS: Failed delete dsc file uuid:{}".format(uuid))
```

Comment:
can we extract any useful info on error from extpiton, or expection ew get here is generic?